templates/verify.ejs

```diff
@@ -1,7 +1,7 @@
-<h1>Thank You</h1>
+<h1>{{Thank You | globalize}}</h1>
 
 <p>
-  Thanks for registering. Please follow the link below to complete your registration.
+  {{Thanks for registering. Please follow the link below to complete your registration. | globalize}}
```

Comment:
Html globalization also depends on the filter.  `globalize` or `translate` might be it.   Work with the html rendering engine developer.